src/statemachine/RSocketStateMachine.cpp

```diff
@@ -62,14 +62,15 @@ RSocketStateMachine::~RSocketStateMachine() {
 
 void RSocketStateMachine::setResumable(bool resumable) {
   debugCheckCorrectExecutor();
-  DCHECK(isDisconnectedOrClosed()); // we allow to set this flag before we are
-  // connected
+  DCHECK(isDisconnectedOrClosed()); // we allow to set this flag only before
```

Comment:
Can you line up these `//` comments?  Maybe just move them above `DCHECK(isDisconnectedOrClosed())` and align with that.